modules/dbnd-airflow/setup.py

```diff
@@ -14,7 +14,6 @@
 requirements_for_airflow = [
     "WTForms<2.3.0",  # fixing ImportError: cannot import name HTMLString at 2.3.0
     "Werkzeug<1.0.0,>=0.15.0",
-    "cattrs==1.0.0",  # airflow requires ~0.9 but it's py2 incompatible (bug)
```

Comment:
please remove from the current MR, let's test it in a separate one  ( we should remove all requirements_for_airflow apparently)